io.catenax.single_level_bom_as_specified/2.0.0/SingleLevelBomAsSpecified.ttl

```diff
@@ -0,0 +1,211 @@
+:createdOn a bamm:Property;
+    bamm:preferredName "Created On"@en;
+    bamm:description "The time the item was created on"@en;
+    bamm:characteristic bamm-c:Timestamp;
+    bamm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime.
+:partGeometry a bamm:Property;
+    bamm:preferredName "Part Geometry"@en;
+    bamm:description "Some parts are intended to be used on either the left or right side, e.g. mirrors, hence this entity describes on which side a part is to be used"@en;
+    bamm:characteristic :PartGeometryCharacteristic;
+    bamm:exampleValue "L - Left".
+:PartQuantityCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Part Quantity Characteristic"@en;
+    bamm:description "The characteristic of the part quantity."@en;
+    bamm:dataType :PartQuantityEntity.
+:PartQuantityEntity a bamm:Entity;
+    bamm:preferredName "Part Quantity Entity"@en;
+    bamm:description "The entity encapsulating the properties describing the quantity of a part."@en;
+    bamm:properties (:measurementUnit :quantityNumber :measurementUnit :quantityNumber).
+:measurementUnit a bamm:Property;
+    bamm:preferredName "Measurement Unit"@en;
+    bamm:description "Unit of measurement for the quantity of serialized objects."@en;
+    bamm:characteristic bamm-c:UnitReference;
+    bamm:exampleValue "kW"^^bamm:curie.
+:quantityNumber a bamm:Property;
+    bamm:preferredName "Quantity Number"@en;
+    bamm:description "The number of objects related to the measurement unit."@en;
+    bamm:characteristic :NumberOfObjects;
+    bamm:exampleValue "350"^^xsd:double.
+:NumberOfObjects a bamm:Characteristic;
+    bamm:preferredName "Number of Objects"@en;
+    bamm:description "Quantifiable number of objects in reference to the measurementUnit."@en;
+    bamm:dataType xsd:double.
+:PartClassificationCharacteristic a bamm-c:Set;
+    bamm:preferredName "Part Classification Characteristic"@en;
+    bamm:description "The characteristic of the part classification."@en;
+    bamm:dataType :PartClassificationEntity.
+:PartClassificationEntity a bamm:Entity;
+    bamm:preferredName "Part Classification Entity"@en;
+    bamm:description "The entity encapsulating the properties of the part classification."@en;
+    bamm:properties (:key :value :key :value).
+:key a bamm:Property;
+    bamm:preferredName "Key"@en;
+    bamm:description "Key within the classification."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "BMW:PartFamily".
+:value a bamm:Property;
+    bamm:preferredName "Value"@en;
+    bamm:description "Value within the classification."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "STEEWHL".
+:PartGeometryCharacteristic a bamm-c:Enumeration;
+    bamm:preferredName "Part Geometry Characteristic"@en;
+    bamm:description "Describes the characteristic of the left or right mark entity with its possible values (left or right)"@en;
+    bamm:dataType xsd:string;
+    bamm-c:values ("L - Left" "R - Right").
```

Comment:
Depending on how the model is supposed to be used, the whitespaces in the enumeration values could be a source of error, if they are used in comparisons.